source/scripts/notesEditor.js

```diff
@@ -15,10 +15,6 @@ async function init() {
     //if preview is not set to true in url
     if(urlArray.length == 2){
         id = urlArray[1];
-    }
-    //if preview is set to true in url
-    if (urlArray.length == 3) {
-        id = urlArray[1].split('?')[0];
         preview = true;
```

Comment:
Since we're not putting preview in the url anymore, we shouldn't have the preview boolean in the code anymore since everything is just dependent on whether id exists or not. The preview boolean adds confusion.